packages/flutter/test/material/button_bar_test.dart

```diff
@@ -5,6 +5,8 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+final Finder _buttonBarRow = find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_ButtonBarRow');
```

Comment:
Please move this finder into the test that is using it.

Reply:
Done!